src/Generating/Trees.cpp

```diff
@@ -58,7 +59,8 @@ static const Vector3d & pickBranchDirection(const cNoise a_Noise, Vector3i a_Blo
 		}
 	};
 
-	size_t index = static_cast<size_t>(a_Noise.IntNoise3DInt(a_BlockPos.x, a_BlockPos.y + a_Seq, a_BlockPos.z)) % directions.size();
+	// Old: size_t index = static_cast<size_t>(a_Noise.IntNoise3DInt(a_BlockPos.x, a_BlockPos.y + a_Seq, a_BlockPos.z)) % directions.size();
```

Comment:
This resulted in many trees having the branches in the same/close direction

Reply:
I just noticed that the function descriptions is not accurate anymore and that the passed values are not used at all